Cargo.toml

```diff
@@ -1,25 +1,37 @@
 [package]
 name = "rush"
 version = "0.2.0"
-authors = ["Cardinal Cryptography"]
 edition = "2018"
+authors = ["Cardinal Cryptography"]
+categories = ["algorithms", "data-structures", "cryptography", "database"]
+documentation = "https://docs.rs/?"
+homepage = "https://alephzero.org"
+repository = "https://github.com/cardinal-cryptography/?"
+keywords = ["aleph", "asynchronous", "consensus", "bft", "blockchain", "distributed-systems"]
+license = "Apache-2.0"
+readme = "README.md"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+description = """
+    Aleph is an asynchronous and Byzantine fault tolerant consensus protocol aimed
+    at ordering arbitrary messages (transactions). It has been designed to operate
+    continuously under conditions where there is no bound on message-delivery delay
+    and under the assumption that there is a significant probability of malicious
+    behavior, making it an excellent fit for blockchain-related applications.
+    """
```

Comment:
It has been designed to continuously operate even in the harshest conditions: with no bounds on message-delivery delays and in the presence of malicious actors. This makes it an excellent fir for blockchain-related applications.

Reply:
Changed.